Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -1284,7 +1284,10 @@ void qMRMLSliceControllerWidgetPrivate::onSliceLogicModifiedEvent()
   // For thin volumes such as ultrasound, the slice bounds may be smaller than the spacing due to floating point errors.
   // To avoid warnings caused by the step size being larger than the slider maximum, set the step size to the smaller of
   // the either the slice offset or the slice bounds depth.
-  offsetResolution = std::min(offsetResolution, sliceBounds[5] - sliceBounds[4]);
+  if (sliceBounds[5] = sliceBounds[4] > 0.0)
```

Comment:
For sake of clarify, what about changing this to:

```
   if (sliceBounds[4] > 0.0)
     {
     sliceBounds[5] = sliceBounds[4]
     [...]
     }
```

Reply:
Whoops, that's actually a typo. Should be:
```if (sliceBounds[5] - sliceBounds[4] > 0.0)```

Thanks for catching it!